### PR TITLE
Fix calltaker map crash when planning scooter trips

### PR DESCRIPTION
**Pull request by demory**
Fixes white-screen-of-death crash when attempting to plan a scooter+transit trip in the calltaker module.

### CODE REVIEW

File: lib/components/map/default-map.js [hunk 1]
```diff
@@ -145,7 +145,7 @@ class DefaultMap extends Component {
           const overlayCompany = oConfig.companies[0] // TODO: handle multi-company overlays
           if (added.includes(overlayMode)) {
             // Company-based mode was just selected; enable overlay iff overlay's company is active
-            if (newQuery.companies.includes(overlayCompany)) {
+            if (newQuery.companies && newQuery.companies.includes(overlayCompany)) {
```
Comment (binh-dam-ibigroup) [inline lib/components/map/default-map.js:148]: You can also use the shorthand notation below:
```suggestion
            if (newQuery.companies?.includes(overlayCompany)) {
```

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Review by miles-grant-ibigroup — APPROVED** [review]
This looks great and is to me the best way to resolve this bug. 

I agree with Binh about the shorthand notation.

**Comment by demory** [comment]
@miles-grant-ibigroup / @binh-dam-ibigroup thanks! Added shorthand per comments. This ok to merge now?

**Comment by miles-grant-ibigroup** [comment]
Just have to approve the percy changes and then we should be good to merge yes!

**Pull request merged**